.gitignore

```diff
@@ -6,6 +6,7 @@
 
 # Ignore bundler config.
 /.bundle
+/vendor
```

Comment:
For what?

Reply:
this is connected with my computer, I started working on the computer at home. And when I installed everything, I created this folder. Andrey said add it to gitignor